packages/react-native/Libraries/Text/Text.js

```diff
@@ -9,110 +9,366 @@
  */
 
 import type {PressEvent} from '../Types/CoreEventTypes';
-import type {TextProps} from './TextProps';
+import type {NativeTextProps} from './TextNativeComponent';
+import type {PressRetentionOffset, TextProps} from './TextProps';
 
-import * as ReactNativeFeatureFlags from '../../src/private/featureflags/ReactNativeFeatureFlags';
 import * as PressabilityDebug from '../Pressability/PressabilityDebug';
 import usePressability from '../Pressability/usePressability';
 import flattenStyle from '../StyleSheet/flattenStyle';
 import processColor from '../StyleSheet/processColor';
 import Platform from '../Utilities/Platform';
 import TextAncestor from './TextAncestor';
 import {NativeText, NativeVirtualText} from './TextNativeComponent';
-import TextOptimized from './TextOptimized';
 import * as React from 'react';
 import {useContext, useMemo, useState} from 'react';
 
+type TextForwardRef = React.ElementRef<
+  typeof NativeText | typeof NativeVirtualText,
+>;
+
 /**
  * Text is the fundamental component for displaying text.
  *
  * @see https://reactnative.dev/docs/text
+ *
+ * NOTE: This file is a short term fork of `Text` for the purpose of performance
+ *       testing. Any changes to either this component or `Text` should be avoided
+ *       while this file exists.
```

Comment:
I guess its not much of the short-term fork anymore if its moved into main Text component. Same with the diffs below.

Reply:
thanks for noticing that! Let me remove it.